pkg/cmd/server/start/start_api.go

```diff
@@ -70,7 +70,6 @@ func NewCommandStartMasterAPI(name, basename string, out io.Writer) (*cobra.Comm
 	// This command only supports reading from config and the listen argument
 	flags.StringVar(&options.ConfigFile, "config", "", "Location of the master configuration file to run from. Required")
 	cmd.MarkFlagFilename("config", "yaml", "yml")
-	// TODO: add health and metrics endpoints on this listen argument
 	BindListenArg(options.MasterArgs.ListenArg, flags, "")
```

Comment:
> I did remove that one... I thought...

This line that I'm commenting on right here says `BindListenArg(options.MasterArgs.ListenArg, flags, "")`.  I thought we agreed that the master api should respect the config file listen args.